client/db/types.go

```diff
@@ -513,11 +513,6 @@ func MatchIsActive(match *order.UserMatch, proof *MatchProof) bool {
 		return false
 	}
 
-	// Refunded matches are inactive regardless of status.
-	if len(proof.RefundCoin) > 0 {
-		return false
-	}
-
```

Comment:
Actually seems ok, I guess because we have separate buckets for active and archived, but something does go wrong if I update while waiting for a refund...